CSS/styles.css

```diff
@@ -0,0 +1,47 @@
+body {
+ 
+    color: rgb(221, 221, 221);
+    background-color: rgb(37, 37, 117);
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+#whole{
+    margin: 50px;
+}
+
+#form{
+    background-color: rgb(36, 35, 35);
+    margin: 20%; 
+    margin-top: 5%;
+    color: white;
+    width: 70%;
+    /*text-align: center; */
+    opacity: 65%;
+}
+#submit{
+   
+    background-color: aqua;
+    width: 100%;
+    margin-top: 5%;
+    padding: 20px;
+}
+
+#comment{
+    width:  100%;
+    height: 120px;
+    color: lightgrey;
+   
+    
+}
+
+header {
+    text-align: center;
+    
+}
+h1 {
+    font-weight: bold;
+}
+#text {
+    margin: 10%;
+    padding-bottom: 100px;
+    padding-top: 40px;
+}
```

Comment:
Kindly make sure to only add two spaces of indentation, to make your code more readable.